tests/test_group/test_billing_group_model.py

```diff
@@ -0,0 +1,79 @@
+from django.core.exceptions import ObjectDoesNotExist
+from django.urls import reverse
+
+from service_catalog.models import BillingGroup
+from tests.test_group.test_group_base import TestGroupBase
+
+
+class TestGroupModel(TestGroupBase):
+
+    def setUp(self):
+        super(TestGroupModel, self).setUp()
+
+    def test_create_billing_group(self):
+        url = reverse('service_catalog:billing_group_create')
+        test_list = [
+            {'data': {'name': 'group_test2'}, 'expected': 2},
+            {'data': {'name': 'billing_group_test2'}, 'expected': 3},
+            {'data': {'name': 'billing_group_test2'}, 'expected': 3},
+            {'data': {'name': ''}, 'expected': 3},
+            {'data': {'foo': 'group'}, 'expected': 3},
+            ]
+        for test in test_list:
+            self.client.post(url, data=test['data'])
+            self.assertEquals(len(BillingGroup.objects.all()), test['expected'])
+
+    def test_edit_billing_group(self):
+        args_group = {
+            'billing_group_id': self.my_billing_group.id
+        }
+        url = reverse('service_catalog:billing_group_edit', kwargs=args_group)
+        test_list = [
+            {'data': {'name': 'a'}, 'expected': 'a'},
+            {'data': {'name': ''}, 'expected': 'a'},
+            {'data': {'name': self.my_billing_group2.name}, 'expected': 'a'},
+            {'data': {'foo': 'group'}, 'expected': 'a'},
+            {'data': {'name': 'b'}, 'expected': 'b'},
+        ]
+        for test in test_list:
+            self.client.post(url, data=test['data'])
+            self.assertEquals(BillingGroup.objects.get(id=self.my_billing_group.id).name, test['expected'])
+
+    def test_delete_billing_group(self):
+        args_group = {
+            'billing_group_id': self.my_billing_group.id
+        }
+        url = reverse('service_catalog:billing_group_delete', kwargs=args_group)
+        self.client.post(url)
+        self.assertRaises(BillingGroup.objects.get(id=self.my_billing_group.id), ObjectDoesNotExist)
```

Comment:
Please switch to self.assertFalse(Object.objects.filter(id=id_to_delete).exists())